.github/actions/build-vsix/action.yml

```diff
@@ -14,5 +14,11 @@ runs:
         npm_config_build_from_source: true
       shell: bash
 
+    # Run again, as the download of zmq binaries could have failed with 403 errors
```

Comment:
could you set the GITHUB_TOKEN in the `npm ci` and then that will be the same as this?

Reply:
I didn't want GITHUB_TOKEN to be accessbiel by all postinstaall steps, that would mean any npm package that has post install steps would now have access to this github token, & I wanted to minimize that.